FLEDGE.md

```diff
@@ -778,7 +838,7 @@ The arguments to `generateBid()` are:
 *   auctionSignals: As provided by the seller in the call to `runAdAuction()`.  This is the opportunity for the seller to provide information about the page context (ad size, publisher ID, etc), the type of auction (first-price vs second-price), and so on.
 *   perBuyerSignals: The value for _this specific buyer_ as taken from the auction config passed to `runAdAuction()`.  This can include contextual signals about the page that come from the buyer's server, if the seller is an SSP which performs a real-time bidding call to buyer servers and pipes the response back, or if the publisher page contacts the buyer's server directly.  If so, the buyer may wish to check a cryptographic signature of those signals inside `generateBid()` as protection against tampering.
 *   trustedBiddingSignals: An object whose keys are the `trustedBiddingSignalsKeys` for the interest group, and whose values are those returned in the `trustedBiddingSignals` request. This used when the trusted server is same-origin with the buyer's script.
-*   crossOriginTrustedSignals: Like turstedBiddingSignals, but used when the trusted-server is
+*   crossOriginTrustedSignals: Like trustedBiddingSignals, but used when the trusted-server is
     cross-origin to the buyer's script. The value is an object that has as a key the trusted
     server's origin, e.g. `"https://example.org"`, and as value an object in format trustedBiddingSignals uses.
```

Comment:
>...a key the trusted server's origin, e.g. `"https://example.org"`

Suggest using the same example value as in the preceding markup for consistency as well as it is clearly an origin.

Reply:
Done